lighthouse-core/test/audits/script-treemap-data-test.js

```diff
@@ -72,6 +74,36 @@ describe('ScriptTreemapData audit', () => {
     });
   });
 
+  describe('coursehero fixture', () => {
+    /** @type {import('../../audits/script-treemap-data.js').TreemapData} */
+    let treemapData;
+    beforeAll(async () => {
+      const context = {computedCache: new Map()};
+      const {map, content} = loadSourceMapFixture('coursehero-bundle-1');
+      expect(map.sourceRoot).toBeTruthy();
+      const mainUrl = 'https://courshero.com';
+      const scriptUrl = 'https://courshero.com/script.js';
+      const networkRecords = [generateRecord(scriptUrl, content.length, 'Script')];
+
+      const artifacts = {
+        URL: {requestedUrl: mainUrl, finalUrl: mainUrl},
+        JsUsage: {},
+        devtoolsLogs: {defaultPass: networkRecordsToDevtoolsLog(networkRecords)},
+        SourceMaps: [{scriptUrl: scriptUrl, map}],
+        ScriptElements: [{src: scriptUrl, content}],
+      };
+      const results = await ScriptTreemapData.audit(artifacts, context);
+
+      // @ts-expect-error: Debug data.
+      treemapData = results.details.treemapData;
+    });
+
+    it('has root nodes', () => {
+      expect(JSON.stringify(treemapData).length).toMatchInlineSnapshot(`31703`);
+      expect(treemapData).toMatchSnapshot();
```

Comment:
Is there *anything* about this we can assert? I'm skeptical of the value of massive snapshots that don't have readable summary statistics. 

Put another way, how will the next reviewer not named @connorjclark decide if changes to this snapshot are good or bad? (especially when it's auto collapsed in GH 😉 )?

Reply:
https://github.com/GoogleChrome/lighthouse/pull/11271#discussion_r500492135

Reply:
I remember :) I accept the canary argument for new functionality we haven't seen in the wild yet and/or don't know how it will change in the future, but this is very specific bug fix PR which I would expect to have a more targeted test. Without *any* assertion it's very difficult for a reviewer to tell if we regressed on this specific point in a +3000/-3000 PR.

Reply:
added an assert for the dupe property